config/environments/production.rb

```diff
@@ -38,8 +37,25 @@
   # config.action_cable.url = "wss://example.com/cable"
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
+
+  # Log to STDOUT by default
+  config.logger = ActiveSupport::Logger.new($stdout)
```

Comment:
How does this play with config/application.rb:54-57? It looks like prior to the upgrade we checked the `RAILS_LOG_TO_STDOUT` env var before defaulting to using stdout

Reply:
https://github.com/alphagov/forms-deploy/pull/395

ah I didn't take it out of https://github.com/alphagov/forms-product-page/blame/main/config/application.rb#L51 which I'll do

Reply:
This seems related to the PR I have open (and just want to check in Dev once more before merging) https://github.com/alphagov/forms-product-page/pull/139

Reply:
We don't have specs for the lograge stuff yet but I'll do that as a separate PR instead of part of the upgrade